telegram/ext/_precheckoutqueryhandler.py

```diff
@@ -43,6 +51,13 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        pattern (:obj:`str` | :func:`re.Pattern <re.compile>` | :obj:`callable` | :obj:`type`, \
+            optional):
+            Pattern to test :attr:`telegram.Update.pre_checkout_query.invoice_payload` against. If
+            a string or a regex pattern is passed, :func:`re.match` is used on
+            :attr:`telegram.Update.pre_checkout_query.invoice_payload` to determine if an update
+            should be handled by this handler.
+
```

Comment:
please add a `.. versionadded:: ` note here. Search for it in the codebase if you want to know how to add them.